src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import cities from "./CityCards";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="container">
+      <div className="row">
+        {cities.map((city, index) => (
```

Comment:
I'm not sure if you talked about this in Group A, but we had a discussion about how it may be useful to further split up this component to show an individual card that uses props to render data.

This way, we can re-use that component all over the application, instead of always having to render multiple cards.

Reply:
yes, it is a good point and I am considering it, thanks Again .

Reply:
yes, it is a good point and I am considering it, thanks Again .